yarn.lock

```diff
@@ -3313,7 +3352,7 @@
     "@types/minimatch" "*"
     "@types/node" "*"
 
-"@types/google-protobuf@^3.10.0", "@types/google-protobuf@^3.7.2":
+"@types/google-protobuf@^3.10.0":
```

Comment:
I thought we have dealt with that. 🤨

Edit: After I ran `yarn` on my laptop on this branch, it reverted back to what this commit removes.

Reply:
Thanks, fixed.